wordpress-mobile/WordPress-Android
# Remove Gutenberg Warning Dialog and related resources

This PR does remove the Gutenberg Warning dialog (that is still shown when opening pages, but not for posts) and its resources.

There is no need to show it, since GB will be the default editor on editing existing GB posts|pages, and the new informative dialog tells the users how to switch back to classic if needed. (Ref: https://github.com/wordpress-mobile/WordPress-Android/pull/9234)

**Test 1**

* Start the app
* Edit a page that was made on the web (or in the mobile apps) by using the old editor
* No Warning dialog on the screen
* Aztec should start (verify that it's Aztec that comes up - look for the "H" toolbar button which only appears in Aztec's case)

(Repeat this for posts)

**Test 2**

* Start the app
* Edit a post that was made with Gutenberg
* Mobile GB should appear and NO warning dialog

(Repeat this for pages)


Update release notes:

- [ x ] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.
